opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -497,42 +497,50 @@ message HistogramDataPoint {
   // buckets if a histogram is provided.
   double sum = 5;
 
-  // bucket_counts is an optional field contains the count values of histogram
-  // for each bucket.
-  //
-  // The sum of the bucket_counts must equal the value in the count field.
+  // buckets is one of several defined bucketing strategies
   //
-  // The number of elements in bucket_counts array must be by one greater than
-  // the number of elements in explicit_bounds array.
-  repeated fixed64 bucket_counts = 6;
+  // If this field is omitted, there is an implied single bucket
+  // spanning (-Inf,+Inf) containing the whole population.
+  oneof buckets {
+    // ExplicitBuckets support encoding arbitrary boundaries and
+    // counts.
+    ExplicitBuckets explicit = 9;
+  }
 
-  // A histogram may optionally contain the distribution of the values in the population.
-  // In that case one of the option fields below and "buckets" field both must be defined.
-  // Otherwise all option fields and "buckets" field must be omitted in which case the
-  // distribution of values in the histogram is unknown and only the total count and sum are known.
+  // (Optional) List of exemplars collected from
+  // measurements that were used to form the data point
+  repeated DoubleExemplar exemplars = 8;
+}
 
-  // explicit_bounds is the only supported bucket option currently.
-  // TODO: Add more bucket options.
+// ExplicitBuckets is a standard way of representing arbitrary
+// histogram data using explicit boundaries.  It combines an
+// array of N-1 boundaries and an array of N counts.
+message ExplicitBuckets {
```

Comment:
Nice to see we revised `histogram` type, wonder do we plan to have any default boundaries or we plan to let each receiver to design themselves?

Reply:
This seems to be a different topic, and I would encourage you to file a different issue for this.

Reply:
Added here: https://github.com/open-telemetry/opentelemetry-proto/issues/280